examples/github/workflows/patcher.yml

```diff
@@ -0,0 +1,26 @@
+name: Patcher
```

Comment:
Is this an example, or an action we want to publish?

Reply:
It's the patcher workflow, not the action